src/test/java/com/google/api/codegen/testdata/nodejs_test_library.baseline

```diff
@@ -565,6 +566,95 @@ describe('LibraryServiceClient', function() {
     });
   });
 
+  describe('streamShelves', function() {
+    it('invokes streamShelves without error', function(done) {
+      var client = libraryV1.libraryServiceClient();
+      // Mock request
+      var request = {};
+
+      // Mock response
+      var expectedResponse = {};
+
+      // Mock Grpc layer
+      client._streamShelves = function() {
+        return new MockStream(request, expectedResponse);
+      };
+
+      var stream = client.streamShelves().on('data', function(response) {
+        assert.deepStrictEqual(response, expectedResponse);
+        done()
+      });
+
+      stream.write(request);
```

Comment:
It seems a bit strange to have a method that returns a server-stream accept a write. Actually, this will [throw](https://github.com/googleapis/gax-nodejs/blob/master/lib/streaming.js#L88) an error.